node/caching-compiler.js

```diff
@@ -103,16 +111,18 @@ function readFileOrNull(filename) {
   }
 }
 
-function getCacheFilename(content, reify) {
-  const hash = createHash("sha1")
-    .update(reifyVersion).update("\0")
-    .update(content).update("\0");
-
-  if (reify) {
-    hash.update(JSON.stringify(reify));
+function getCacheFilename() {
+  const args = [];
+  const argc = arguments.length;
+  for (let i = 0; i < argc; ++i) {
+    args.push(arguments[i]);
```

Comment:
Since this is a straight assignment you can do:
```js
const argsCount = arguments.length;
const args = new Array(argsCount)

for (let i = 0; i < argsCount; ++i) {
  args[i] = arguments[i];
}
```